### DISCUSSION TITLE
Cypress testing AE v4 beta has been released, with Qv2 support

### DISCUSSION THREAD

**Discussion by IlCallo** [quasarframework/quasar#9539] [Announcements]
Hi everyone!
A new major version for Cypress testing AE has been released as beta, notable changes:
- support Qv2, drop Qv1
- updated Cypress to 7.x
- Quasar helper commands and related types are now exported directly from the package

See release notes here: https://github.com/quasarframework/quasar-testing/releases/tag/4.0.0-beta.1

**Comment by ontwikkelfabriek** [comment]
@IlCallo

So you can still use v3 for Qv1?

**Reply by IlCallo** [reply]
Yes, of course, and we'll provide bugfixes for v3 until Qv1 will reach EOL in April 2022.
But new features and Cypress updates will be applied to v4 only
